appveyor.yml

```diff
@@ -1,38 +1,38 @@
----
-version: "{build}"
+version: '{build}'
 branches:
```

Comment:
Yes, we have in the past and might need to in the future, especially for security-related issues.